code/espurna/thermostat.ino

```diff
@@ -304,7 +305,7 @@ void _thermostatWebSocketOnSend(JsonObject& root) {
   root["thermostatVisible"] = 1;
   root[NAME_TEMP_RANGE_MIN] = _temp_range.min;
   root[NAME_TEMP_RANGE_MAX] = _temp_range.max;
-  root[NAME_REMOTE_SENSOR_NAME] = _thermostat.remote_sensor_name;
```

Comment:
The idea is that user in web interface set remote sensor name and this is enough to compose topic. And user can see remote sensor name i web interface. Removing this string woud affect web interface - user would not able to see current remote sensor name.